requirements.txt

```diff
@@ -1,10 +1,3 @@
 ansible==9.5.1
-cryptography==42.0.7
-jinja2==3.1.4
-jmespath==1.0.1
```

Comment:
Per CI error message, this is needed for `json_query`

Reply:
Jinja2 and crypto are downloaded by ansible-core, added only jmespath.
ready to retest.